lib/js/index.js

```diff
@@ -67,20 +72,23 @@ class DateTimePicker extends Events {
   // style options
   initializeRome(container, validator) {
     const onData = this.onChangeDate.bind(this);
-
+    const weekDays = moment.localeData(this.options.locale)._weekdaysMin;
```

Comment:
Hey there!

In JS there is a convention that indicates variables starting with underscore are private (Futher reading [here](https://www.thoughtco.com/and-in-javascript-2037515?i10c) or [here](https://stackoverflow.com/questions/8288756/in-javascript-what-does-this-underscore-mean)) They should not be read or written directly. It's just a convention but that's why I decided to update moment to 2.13.0.

Reply:
@Luisetelo this solution was used due to the version of moment.js this project uses.